### DISCUSSION TITLE
Faiss >1.6.1

### DISCUSSION THREAD

**Discussion by hadim** [milvus-io/milvus#5106] [Q&A and General discussion] [answered]
We are considering using Milvus for an internal project. We noticed milvus uses an old version of faiss (1.6.1) and were wondering if you are planning to keep track of upstream changes in faiss (by regularly syncing to the latest version for example)?

We don't have any specific needs for faiss >1.6.1 at the moment but we are trying to anticipate future needs.

**Comment by shengjun1985** [comment, marked as answer]
Now, Milvus (v1.0, v1.1) is using Faiss 1.6.3.
Of cause we will track the upstream changes.
But Faiss 1.7.0 changed its architecture a lot, so it will take a while.
However, we are doing it right now.
